### PR TITLE
Update the hugo theme for mobile screens

### PR DESCRIPTION
**Pull request by sodevious**
Addresses #37 mobile issues and #38

### CODE REVIEW

File: hugo_site/themes/dceu2019/static/css/style.css [hunk 1]
```diff
@@ -1,23 +1,32 @@
...
+.header_image a {
+  max-width: 600px;
+  margin: 0 auto;
+  display: block;
+}
+
+.header_image img {
+  width: 100%;
+  margin: 0 auto;
+}
```
Comment (benjaoming) [inline hugo_site/themes/dceu2019/static/css/style.css]: This part makes the header image stretch beyond its size. Notice the conditional statement in the template that makes the logo big on the front page and smaller on the other pages.

image

**Comment by benjaoming** [comment]
Thanks for the flex-box update, that's nice!

Do you have a quickfix for the logo such that it only scales up to it's maximum resolution and doesn't go beyond? Perhaps a `max-width: 100%` would fix both versions (large on front page and small on other pages)?

**Comment by sodevious** [comment]
@benjaoming pushed a fix that should address both logo versions!

**Comment by benjaoming** [comment]
Yes, works here now, thanks!

**Pull request merged**